### PR TITLE
Fix/screen sharing audio

### PR DESCRIPTION
**Pull request by jordanreedie**
Fix missing audio during screen share

## Description

- Add peers' audio streams to the DOM when someone else is screen sharing.
- Some code style changes to make stuff more readable
- Change browser warning
- Also changed default height for remote video containers (< 4 peers) from 75vh to 80vh to better use screen space

## How has this been tested?

Tested locally. This is really difficult to test without other people / machines, but I think it works.

## Types of changes

- ✅ Bug fix (non-breaking change which fixes an issue)

## Checklist:

- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ All new and existing tests passed. **[REQUIRED]**

### CODE REVIEW

File: src/components/Chat/PeerVideo.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+class PeerVideo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.appendVideo = this.appendVideo.bind(this);
+    this.video = this.props.videoEl;
+  }
+
+  appendVideo(el) {
+    logger.debug("appending?", "color:", this.props.peerColor)
```
Comment (mlippert) [inline src/components/Chat/PeerVideo.js]: For our style consistency, js strings are single quoted (unless a overriding reason like quoting a single quote) and double quotes for html (in jsx) strings? does that seem like a reasonable rule? Would we rather just prefer one or the other (I think I slightly prefer single quotes) everywhere. This isn't really a review comment for this particular PR, just starting the discussion.

File: src/components/Chat/SharedScreen.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { logger } from '../../libs/utils';
+import PeerVideo from './PeerVideo'
```
[the remark this answers is not in the document]
Comment (jordanreedie) [reply src/components/Chat/SharedScreen.js]: whoops, just missed it on those ones, thanks

File: src/components/Chat/PeerVideo.js (hunk 1 above)
Comment (jordanreedie) [reply src/components/Chat/PeerVideo.js]: oh yeah, i agree, single quotes in JS and double in html. my bad :)

File: src/components/Chat/SharedScreen.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mlippert) [reply src/components/Chat/SharedScreen.js]: yep I figured, that's why even a quick once over by a 2nd set of eyes helps :-)

File: src/redux/reducers/index.js [hunk 3]
```diff
@@ -25,8 +25,9 @@ const chatPersistConfig = {
   key: 'chat',
   storage: storage,
   blacklist: ['webRtcPeers', 'webRtcRiffIds', 'volume', 'roomName',
-              'inRoom', 'joiningRoom', 'getMediaError',
-              'savedDisplayName']
+              'inRoom', 'joiningRoom', 'getMediaError', 'audioMuted',
```
Comment (jordanreedie) [inline src/redux/reducers/index.js]: noting that this fixes https://trello.com/c/8Idkk0r8/60-when-you-leave-a-meeting-muted-and-then-you-come-back-to-any-meeting-you-stay-muted-with-no-warning

**Comment by mlippert** [comment]
[When you leave a meeting muted, and then you come back to any meeting, you stay muted with no warning](https://trello.com/c/8Idkk0r8/60-when-you-leave-a-meeting-muted-and-then-you-come-back-to-any-meeting-you-stay-muted-with-no-warning)

**Pull request merged**